src/tasks/tasks.py

```diff
@@ -359,11 +361,13 @@ def process_split(self, split, indexers) -> Iterable[Type[Instance]]:
         ''' Process split text into a list of AllenNLP Instances. '''
         return process_single_pair_task_split(split, indexers, is_pair=True,
                                               classification=False)
+
     def update_metrics():
         # currently don't support metrics for regression task
-        # TODO(Yada): support them! 
+        # TODO(Yada): support them!
```

Comment:
Make an issue.

Reply:
Or explain here why this isn't really needed. (It might not be!)

Reply:
Done